packages/core/test/__snapshots__/Breadcrumbs.js.snap

```diff
@@ -73,11 +57,8 @@ exports[`Breadcrumbs renders basic 1`] = `
   >
     <a
       className="c2"
-      color="text.light"
-      href="https://www.priceline.com/flights/"
```

Comment:
Here is an example of the carnage that would ensue if my other comment isn't addressed 😛

Reply:
Haha, that'll do it 😅 Fixed!